test/common/testlib.py

```diff
@@ -2540,13 +2549,17 @@ def wait(func: Callable, msg: Optional[str] = None, delay: int = 1, tries: int =
     raise Error(msg or "Condition did not become true.")
 
 
-def sit(machines={}):
+def sit(machines=None):
     """
     Wait until the user confirms to continue.
 
     The current test case is suspended so that the user can inspect
     the browser.
     """
+    if machines is None:
+        sys.stderr.write("No machines found to sit in")
```

Comment:
Needs a `\n`. But moreover, I think that'd make more sense as an `assert`?

Reply:
Changed it to an assert, that's a good suggestion.